### DISCUSSION TITLE
作者真厉害

### DISCUSSION THREAD

**Discussion by NebulaTechs** [Meloong-Git/PCL#2246] [讨论]
做了两坤年软件还是没有DLL(OCX)文件。[鼓掌]

**Comment by NebulaTechs** [comment]
[Plain Craft Launcher 2 setup.zip](https://github.com/Hex-Dragon/PCL2/files/12046836/Plain.Craft.Launcher.2.setup.zip)
对了作者为什么不做一个安装包？我已经做好了。点击上方即可下载。

**Reply by NebulaTechs** [reply]
注意，这是正式版，不是预发布版。

**Comment by WorldHim** [comment]
dll不一定要附带在可执行文件同目录下，系统目录里的都可直接使用。。
而且就一个文件还有必要做安装包吗。。。

**Reply by iPodtouch6557** [reply]
不做安装包我个人认为虽然会在某些地方会欠缺，但是大大提升了简洁性和美观性，还是很好的，~毕竟谁也不想看见自己电脑里一堆安装包还得自己清~

**Reply by WorldHim** [reply]
但是单文件没必要做安装包。。。

**Reply by iPodtouch6557** [reply]
是的（
我认为单文件做这玩意就很不简洁，不建议做，~做了难道拿来装13吗~

**Reply by NebulaTechs** [reply]
不，项目越做越大，早晚系统的dll要不够用的。除非能在临时目录生成，不然肯定有dll。

**Reply by PlayMcBKuwu** [reply]
其实可以像曾经的 **360 文件粉碎机 独立版** 一样做出类自解压（雾）
